libs/ui-lib/lib/common/config/constants.ts

```diff
@@ -364,3 +364,6 @@ export const CHANGE_ISO_PASSWORD_FILE_LINK =
 
 export const CUSTOM_MANIFESTS_HELP_LINK =
   'https://docs.openshift.com/container-platform/4.12/installing/install_config/installing-customizing.html';
+
+export const HOW_TO_KNOW_IF_CLUSTER_SUPPORTS_MULTIPLE_CPU_ARCHS =
```

Comment:
Please add a `// TODO(mortegag)` mentioning the MGMT ticket and that this link will need to be changed in the future